torch/csrc/jit/tracer_state.h

```diff
@@ -64,7 +64,7 @@ struct TracingState : public std::enable_shared_from_this<TracingState> {
   // TODO: Perhaps, turn this into an owning reference.  The buffers
   // are persistent, so this won't lead to a leak.
   std::unordered_map<void*, Node*> buffer_map;
-  std::vector<std::vector<VariableFlags>> var_flags;
+  std::vector<std::pair<std::vector<VariableFlags>, std::vector<VariableFlags>>> var_flags;
```

Comment:
What is the first VariableFlags, and what is the second one? (Comment preferred.)

Reply:
Input/output flags